### PR TITLE
[zh]Add 2021-12-08-dual-stack-networking-ga.md

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fix: Issues:#32409
[zh]Add 2021-12-08-dual-stack-networking-ga.md

### CODE REVIEW

File: content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+Starting in 1.20 with the re-implementation of dual-stack services as alpha, the underlying networking for Kubernetes has included dual-stack whether or not a cluster was configured with the feature flag to enable dual-stack.
+-->
+从 1.20 版本开始，alpha 版本重新实现了双协议栈服务，无论集群是否配置了启用双协议栈的特性标志，
+Kubernetes 的底层网络都已经包括了双协议栈。
+
+<!--
+Kubernetes 1.23 removed that feature flag as part of graduating the feature to stable. Dual-stack networking is always available if you want to configure it. You can set your cluster network to operate as single-stack IPv4, as single-stack IPv6, or as dual-stack IPv4/IPv6.
+-->
+Kubernetes 1.23 删除了这个特性标志，说明该特性已经稳定。
+如果你想要配置双协议栈网络，它总是可用的。你可以将集群网络设置为 IPv4 单协议栈 、IPv6 单协议栈或 IPV4/IPV6 双协议栈 。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md:77]:
```suggestion
如果你想要配置双协议栈网络，这一能力总是存在的。
你可以将集群网络设置为 IPv4 单协议栈、IPv6 单协议栈或 IPV4/IPV6 双协议栈 。
```

File: content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+## Try dual-stack right now
+-->
+## 现在尝试双协议栈
+
+<!--
+While upstream Kubernetes now supports [dual-stack networking](/docs/concepts/services-networking/dual-stack/) as a GA or stable feature, each provider’s support of dual-stack Kubernetes may vary. Nodes need to be provisioned with routable IPv4/IPv6 network interfaces. Pods need to be dual-stack. The [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) is what assigns the IP addresses to the Pods, so it's the network plugin being used for the cluster that needs to support dual-stack. Some Container Network Interface (CNI) plugins support dual-stack, as does kubenet.
+-->
+虽然现在上游 Kubernetes 支持[双协议栈网络](/zh/docs/concepts/services-networking/dual-stack/)
+作为 GA 或稳定特性，但每个提供商对双协议栈 Kubernetes 的支持可能会有所不同。节点需要提供可路由的 IPv4/IPv6 网络接口。
+Pod 需要是双协议栈的。[网络插件](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md:103]:
```suggestion
Pod 需要是双协议栈的。[网络插件](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
```

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

File: content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+是用来为 Pod 分配 IP 地址的，所以集群需要支持双协议栈的网络插件。一些容器网络接口（CNI）插件支持双协议栈，例如 kubenet。
+
+<!--
+Ecosystem support of dual-stack is increasing; you can create [dual-stack clusters with kubeadm](/docs/setup/production-environment/tools/kubeadm/dual-stack-support/), try a [dual-stack cluster locally with KIND](https://kind.sigs.k8s.io/docs/user/configuration/#ip-family), and deploy dual-stack clusters in cloud providers (after checking docs for CNI or kubenet availability).
+-->
+支持双协议栈的生态系统在不断增加；你可以使用 [kubeadm 创建双协议栈集群](/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support/),
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md:109]:
```suggestion
支持双协议栈的生态系统在不断壮大；你可以使用
[kubeadm 创建双协议栈集群](/zh/docs/setup/production-environment/tools/kubeadm/dual-stack-support/)，
```

File: content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+在本地尝试用 [KIND 创建双协议栈集群](https://kind.sigs.k8s.io/docs/user/configuration/#ip-family)，
+并将双协议栈集群部署到云上（在查阅 CNI 或 kubenet 可用性的文档之后）
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md:111]:
```suggestion
还可以将双协议栈集群部署到云上（在查阅 CNI 或 kubenet 可用性的文档之后）
```

File: content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+<!--
+## Get involved with SIG Network
+-->
+## 加入 Network SIG
+
+<!--
+SIG-Network wants to learn from community experiences with dual-stack networking to find out more about evolving needs and your use cases. The [SIG-network update video from KubeCon NA 2021](https://www.youtube.com/watch?v=uZ0WLxpmBbY&list=PLj6h78yzYM2Nd1U4RMhv7v88fdiFqeYAP&index=4) summarizes the SIG’s recent updates, including dual-stack going to stable in 1.23.
+-->
+SIG-Network 希望你从双协议栈网络的社区体验中学习，以了解更多不断变化的需求和你的用例信息。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md:121]:
```suggestion
SIG-Network 希望从双协议栈网络的社区体验中学习，以了解更多不断变化的需求和你的用例信息。
```

File: content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+[SIG-network 更新了来自 KubeCon 2021 北美大会的视频](https://www.youtube.com/watch?v=uZ0WLxpmBbY&list=PLj6h78yzYM2Nd1U4RMhv7v88fdiFqeYAP&index=4)
+总结了 SIG 最近的更新，包括双协议栈将在 1.23 版本中稳定。
+
+<!--
+The current SIG-Network [KEPs](https://github.com/orgs/kubernetes/projects/10) and [issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+label%3Asig%2Fnetwork) on GitHub illustrate the SIG’s areas of emphasis. The [dual-stack API server](https://github.com/kubernetes/enhancements/issues/2438) is one place to consider contributing.
+-->
+当前 SIG-Network 在GitHub上的 [KEPs](https://github.com/orgs/kubernetes/projects/10) 和
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-12-08-dual-stack-networking-ga.md:128]:
```suggestion
当前 SIG-Network 在 GitHub 上的 [KEPs](https://github.com/orgs/kubernetes/projects/10) 和
```

**Comment by zhangxyjlu** [comment]
I have revised all the review comments. Thank you@[tengqm](https://github.com/tengqm)

**Comment by howieyuen** [comment]
please squash your commits into one.

**Comment by zhangxyjlu** [comment]
> please squash your commits into one.

I have rebased it. Thanks for your review.@[howieyuen](https://github.com/howieyuen)

**Comment by howieyuen** [comment]
Thanks~
/lgtm

**Pull request merged**